examples/v0/reporting/get_hotel_ads_performance.py

```diff
@@ -0,0 +1,89 @@
+import sys
+
+import google.ads.google_ads.client
+
+
+_DEFAULT_PAGE_SIZE = 50
+
+
+def main(client, customer_id, page_size):
+    ga_service = client.get_service('GoogleAdsService')
+
+    query = ('SELECT campaign.id, campaign.advertising_channel_type, '
+             'ad_group.id, ad_group.status, metrics.impressions, '
+             'metrics.hotel_average_lead_value_micros, '
+             'segments.hotel_check_in_day_of_week, '
+             'segments.hotel_length_of_stay '
+             'FROM hotel_performance_view '
+             'WHERE segments.date DURING LAST_7_DAYS '
+             'AND campaign.advertising_channel_type = \'HOTEL\' '
+             'AND ad_group.status = \'ENABLED\' '
+             'ORDER BY metrics.impressions DESC '
+             'LIMIT 50')
+
+    response = ga_service.search(customer_id, query, page_size=page_size)
+
+    try:
+        for row in response:
+            campaign = row.campaign
+            ad_group = row.ad_group
+            hotel_check_in_day_of_week = row.hotel_check_in_day_of_week
+            hotel_length_of_stay = row.hotel_length_of_stay
+            metrics = row.metrics
+
+            print('Ad group ID "%s" in campaign ID "%s" with hotel check-in '
+                  'on "%s" and "%s" day(s) stay had %d impression(s) and %d '
+                  'average lead value (in micros) during the last 7 days.'
+                  % (ad_group.id.value, campaign.id.value,
+                     hotel_check_in_day_of_week,
+                     hotel_length_of_stay.value, metrics.impressions.value,
+                     metrics.hotel_average_lead_value_micros.value))
+    except google.ads.google_ads.errors.GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
+              'following errors:' % (ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "%s".' % error.message)
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: %s' % field_path_element.field_name)
+        sys.exit(1)
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = (google.ads.google_ads.client.GoogleAdsClient
+                         .load_from_storage())
+
+    parser = argparse.ArgumentParser(
+        description=('Retrieves a campaign\'s negative keywords.'))
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=six.text_type,
+                        required=True, help='The Google Ads customer ID.')
+    parser.add_argument('-p', '--page_size', type=int, required=False,
+                        help='The number of entries to include per page.\
+                        Default is 50.', default=_DEFAULT_PAGE_SIZE)
```

Comment:
Don't hardcode the value in the message. Either remove it or use _DEFAULT_PAGE_SIZE to fill in the number

Reply:
or better yet, remove pagesize as a parameter entirely as it is orthogonal to this example

Reply:
I'll just remove it - good point.